src/saml.ts

```diff
@@ -782,6 +783,10 @@ class SAML {
         validSignature = true;
       }
 
+      if (this.options.wantDocumentSigned === true && validSignature === false) {
```

Comment:
I do not use javascript/typescript so I have to ask this:

Lets assume this typescript library (`passport-saml`) is used inside application implemented with javascript.

Is it possible to configure `SamlStrategy` with
```javascript
...
wantDocumentSigned: "True",
...
```
and if so is it possible that even after all the code generated by typescript to javascript compiler (which may or may not introduce some type validations) this `"True"` ends up to `this.option.wantDocumentSigned`?

If it is possible (and since application developer clearly wanted to use value `true`) would it be safer to use this `if` statement:
```javascript
if (this.options.wantDocumentSigned !== false && validSignature !== true) {
```
This way all truthy (including `"True"`) values would be interpreted to "safer fallback" instead of unsafer.

Reply:
That is a good idea. I'm not a big fan of using negations where possible, but it makes sense here. I've pushed up a PR to handle all such cases in the code-base: https://github.com/node-saml/node-saml/pull/85.